### DISCUSSION TITLE
Does storybook/react-native 7.6.18 work with react native 0.74?

### DISCUSSION THREAD

**Discussion by DannyRoberts22** [storybookjs/storybook#29609] [Help]
### Summary

After upgrading RN from 0.72 to 0.74 Storybook v 7.6.18 has stopped working.

We used to run it with `react-native run-ios --configuration Storybook` after bumping RN to 0.74 it is now returning an error saying `[--configuration` is undefined](error: unknown option '--configuration')`. 

So I set it to run storybook automatically in index.js and it is now throwing an error `[Error: Couldn't find any stories in your Storybook.
- Please check your stories field of your main.js config.
- Also check the browser console and terminal for error messages.]`

I've read up about it and apparently in RN 0.74 the configuration flag is still valid but Architecturally RN behaves differently which I believe is the issue.

So all in all, is it worth trying to fix or should I bump to Storybook 8.3?

### Additional information

 LOG  Storybook: could not find persisted story
 ERROR  [Error: Couldn't find any stories in your Storybook.
- Please check your stories field of your main.js config.
- Also check the browser console and terminal for error messages.]
 LOG  [Error: Couldn't find any stories in your Storybook.
- Please check your stories field of your main.js config.
- Also check the browser console and terminal for error messages.]

**Comment by dannyhw** [comment]
Yes it should work. Those errors seem more like mis configuration just at first glance. Can you share more of your setup?

If you can make a reproduction of the problem that would great
